src/Compilers/CSharp/Portable/Symbols/NullableContextFlags.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// Used by symbol implementations to represent the byte value of [NullableContext].
+    /// </summary>
+    [Flags]
+    internal enum NullableContextFlags : byte
+    {
+        None = 0,
+        Oblivious = 1,
+        NotAnnotated = 2,
```

Comment:
>NotAnnotated [](start = 8, length = 12)

The enum is marked [Flags], is it valid to combine this value with the previous one and what would that mean?